packages/app/components/document/document-header.tsx

```diff
@@ -9,30 +13,112 @@ import ModelIcon from '../model-icon'
 import styles from './document-header.module.css'
 import DocumentStateBadge from './document-state-badge'
 
+type PropsType = {
+    activePanel?: string
+    document?: Document
+    isJsonPanelOpen?: boolean
+    model?: ModelWithWorkflow
+    version?: any
+    toggleJsonDiffer?: () => void
+    togglePanelOpen?: (panel: string) => void
+    privileges?: string[]
+    comments?: DocumentComment[]
+    history?: DocumentHistory[]
+    collaborators?: Collaborator[]
+}
+
 const DocumentHeader = ({
     activePanel = null,
     document = null,
     isJsonPanelOpen = false,
     model,
     version = null,
-    toggleJsonDiffer,
-    togglePanelOpen,
+    toggleJsonDiffer = () => {},
```

Comment:
This is basically a bit of a guard against not knowing if we've passed these functions in. Since adding the `PropsType` above, I wasn't able to get the app to pass the production ESLint check / `tsc` check without marking the toggle fns optional. On the new document page, we now do not pass in these undefined functions.